code/modules/projectiles/guns/shotguns.dm

```diff
@@ -665,6 +670,46 @@ can cause issues with ammo types getting mixed up during the burst.
 		pump_lock = FALSE //we're operating the slide release to unload, thus unlocking the pump
 	return ..()
 
+//***********************************************************
+// Martini Henry
+
+/obj/item/weapon/gun/shotgun/double/martini
+	name = "\improper Martini Henry lever action rifle"
+	desc = "A crowd favorite for anti-Zulu operations."
```

Comment:
id prefer not to reference real life wars like this, rewrite this
Preferably with a reason why its used to hunt xenos

Reply:
ok yeah thats what i figured, ill ask Ciel for another one